tim-refresh/src/main/java/com/trihydro/timrefresh/TimRefreshController.java

```diff
@@ -9,45 +10,49 @@
 import com.google.gson.Gson;
 import com.trihydro.library.helpers.EmailHelper;
 import com.trihydro.library.helpers.TimGenerationHelper;
-import com.trihydro.library.helpers.Utility;
 import com.trihydro.library.model.Logging_TimUpdateModel;
 import com.trihydro.library.model.ResubmitTimException;
 import com.trihydro.library.model.TimUpdateModel;
 import com.trihydro.library.service.ActiveTimService;
 import com.trihydro.timrefresh.config.TimRefreshConfiguration;
 
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
 
 @Component
+@Slf4j
 public class TimRefreshController {
     private final SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
     public Gson gson = new Gson();
     protected TimRefreshConfiguration configuration;
-    private Utility utility;
-    private EmailHelper emailHelper;
-    private ActiveTimService activeTimService;
-    private TimGenerationHelper timGenerationHelper;
+    private final EmailHelper emailHelper;
+    private final ActiveTimService activeTimService;
+    private final TimGenerationHelper timGenerationHelper;
 
     @Autowired
-    public TimRefreshController(TimRefreshConfiguration configurationRhs, Utility _utility,
-            ActiveTimService _activeTimService, EmailHelper _emailHelper, TimGenerationHelper _timGenerationHelper) {
-        configuration = configurationRhs;
-        utility = _utility;
-        activeTimService = _activeTimService;
-        emailHelper = _emailHelper;
-        timGenerationHelper = _timGenerationHelper;
+    public TimRefreshController(TimRefreshConfiguration configurationRhs, ActiveTimService activeTimService, EmailHelper emailHelper,
+                                TimGenerationHelper timGenerationHelper) {
+        this.configuration = configurationRhs;
+        this.activeTimService = activeTimService;
+        this.emailHelper = emailHelper;
+        this.timGenerationHelper = timGenerationHelper;
     }
 
-    @Scheduled(cron = "${cron.expression}") // run at 1:00am every day
+    /**
+     * This method is intended to be run once a day, but can be configured to run at
+     * any time. It will check for Active TIMs that are expiring within 24 hours and
+     * issue new TIMs to the ODE for those that are expiring.
+     */
+    @Scheduled(cron = "${cron.expression}")

```

Comment:
Note: The single-line comment was replaced with more detailed JavaDocs, clarifying the intention to run the task once a day without the risk of misidentifying the time.